### PR TITLE
Minor style fixes

### PR DESCRIPTION
**Pull request by dkfellows**
Cleans up after the data transfer branches.

See also https://github.com/SpiNNakerManchester/SpiNNFrontEndCommon/pull/537

### CODE REVIEW

File: SpiNNaker-front-end/src/main/java/uk/ac/manchester/spinnaker/front_end/NoDropPacketContext.java [hunk 1]
```diff
@@ -164,20 +161,19 @@ public NoDropPacketContext(Transceiver txrx,
 	 * @param monitorCoreLocations
 	 *            The extra monitor cores on the SpiNNaker system that control
 	 *            the routers.
-	 * @param gather
-     *          The gatherer for this context and linked to these
-     *           extra monitor cores.
+	 * @param gatherers
```
Comment (alan-stokes) [inline SpiNNaker-front-end/src/main/java/uk/ac/manchester/spinnaker/front_end/NoDropPacketContext.java]: same here i think

File: SpiNNaker-front-end/src/main/java/uk/ac/manchester/spinnaker/front_end/NoDropPacketContext.java [hunk 2]
```diff
@@ -214,14 +208,13 @@ private static CoreSubsets convertToCoreSubset(
...
-
-	    log.info("switching board at {} to standard mode", firstChip);
+		log.info("switching board at {} to standard mode", firstChip);
 		quietlySetTemporaryTimeouts();
 
 		try {
 			// Do the real reset
-			txrx.setReinjectionTimeout(this.gatherer, lastStatus);
-			txrx.setReinjectionEmergencyTimeout(this.gatherer, lastStatus);
+			txrx.setReinjectionTimeout(gatherers, lastStatus);
```
Comment (alan-stokes) [inline SpiNNaker-front-end/src/main/java/uk/ac/manchester/spinnaker/front_end/NoDropPacketContext.java]: same

File: SpiNNaker-front-end/src/main/java/uk/ac/manchester/spinnaker/front_end/NoDropPacketContext.java [hunk 3]
```diff
@@ -70,26 +71,24 @@
 	 * @param monitorCores
 	 *            The extra monitor cores on the SpiNNaker system that control
 	 *            the routers.
-	 * @param gatherer
-	 *         The gatherer core on the SpiNNaker system that supports the
-	 *         MC control api.
+	 * @param gatherers
```
[the remark this answers is not in the document]
Comment (dkfellows) [reply SpiNNaker-front-end/src/main/java/uk/ac/manchester/spinnaker/front_end/NoDropPacketContext.java]: The cardinality of the names is correct. The advisability of taking advantage of that…

File: SpiNNaker-front-end/src/main/java/uk/ac/manchester/spinnaker/front_end/NoDropPacketContext.java (hunk 1 above)
Comment (dkfellows) [reply SpiNNaker-front-end/src/main/java/uk/ac/manchester/spinnaker/front_end/NoDropPacketContext.java]: same here

File: SpiNNaker-front-end/src/main/java/uk/ac/manchester/spinnaker/front_end/NoDropPacketContext.java (hunk 2 above)
Comment (dkfellows) [reply SpiNNaker-front-end/src/main/java/uk/ac/manchester/spinnaker/front_end/NoDropPacketContext.java]: same

**Comment by dkfellows** [comment]
Updated the comments, but not in the way you were envisaging.

**Comment by dkfellows** [comment]
Passes integration test! 😁

**Pull request merged**